test/repository.spec.js

```diff
@@ -240,6 +240,16 @@ describe('Repository', function() {
             done();
          }));
       });
+      
+      it('should list review comments', function(done) {
+        const repo = github.getRepo('michael', 'github');
```

Comment:
You'll also want to update based on the latest changes from master that got the tests working again.